src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -302,6 +331,17 @@ export class OsdListComponent implements OnInit {
     });
   }
 
+  crushAction() {
+    const selectedOsd = this.osds.filter((o) => o.id === this.selection.first().id).pop();
```

Comment:
Alternatively use lodash.
```suggestion
    const selectedOsd = _.filter(this.osds, ['id', this.selection.first().id]).pop();
```

Reply:
Done

Reply:
Done